### ISSUE TITLE
Off centre presentation of titles on some browsers (web)

### ISSUE THREAD

**Issue by SMHall94** [deep-learning-indaba/Baobab#768]
![Off centre](https://user-images.githubusercontent.com/43542098/92277738-f00bf300-eef3-11ea-930b-ab8648f72141.png)

**Comment by avishkar58** [comment]
Can you confirm which browser(s) this is happening on?

**Comment by SMHall94** [comment]
Yes - Manjaro operating system with a Mozilla Firefox browser. This is the only instance reported out of 8 testers

**Comment by SMHall94** [comment]
I just tested this on Firefox on Ubuntu and it has the same off centre presentation of the "Welcome to Baobab". It works fine in Chrome

**Comment by avishkar58** [comment]
This still occurs on a number of pages, still need to go through all pages in Firefox to see which ones and ideally update some global style to correct it. At the very least, the centered and left aligned components should be consistent between Chrome and Firefox
